### ISSUE TITLE
Hardcoded redmine address in search results

### ISSUE THREAD

**Issue by frogec** [Restream/redmine_elasticsearch#43]
Hi,

I am using bitnami Redmine 3.3.2-2 with current master release of the plugin. Search functionality works without a problem and it's really helpful. 

I found one problem with web address of search results. When I click on a search result I am redirected to address shown in bottom left corner of the picture. 

![screenshot_71](https://cloud.githubusercontent.com/assets/15109727/23614071/06dea238-0282-11e7-85c1-62d4340a5301.png)

My redmine is at: http://localhost:81/redmine, but link on a search result redirects me to http://localhost:3000/.

Is there a way I can override default configuration and set the correct address of Redmine?

Thanks,
Nejc

**Comment by nodecarter** [comment]
Hi @frogec ,
Check your configuration Administration->Settings->General->Host name and path

**Comment by frogec** [comment]
Thank you for your answer. I've set the correct host name and path with your solution.

Because I use Bitnami Redmine stack, I also had to set default Bitnami path to "/" instead of "/redmine".

I used solution provided in this topic: https://community.bitnami.com/t/how-can-change-the-default-url-to-access-my-application/3/2

I executed command in the picture and search results work now.

![image](https://cloud.githubusercontent.com/assets/15109727/23648653/e03143e4-031b-11e7-97ec-8de5bb0466e6.png)

I am closing the Issue.

Have a nice day, 
Nejc
